### ISSUE TITLE
Issues when trying to execute the Appimage

### ISSUE THREAD

**Issue by edgvrl** [domferr/Linux-PowerToys#9]
When trying to execute the Appimage it wont start and spits out this error:
```
 ❯ ./LinuxPowerToys-0.8-x86_64.AppImage

(linuxpowertoys:103670): Gtk-WARNING **: 12:32:14.401: Could not load a pixbuf from /org/gtk/libgtk/icons/16x16/status/window-minimize-symbolic.symbolic.png.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
zsh: IOT instruction (core dumped)  ./LinuxPowerToys-0.8-x86_64.AppImage
```

**Comment by martijn10kb** [comment]
I have the same issue
I `chmod a+x` it and then started it in the gui (outside of the cli), thenit worked

**Comment by domferr** [comment]
Thank you @martijn10kb for sharing a solution!

**Comment by omarzk1308** [comment]
I tried this but not working :(

**Comment by KidesLeo** [comment]
Hit this same crash on CachyOS (Arch-based) with the 0.9 AppImage and dug into the root cause, since the workarounds above didn't explain what's going on.

TL;DR: the app itself is fine. The appimage-builder-generated AppRun environment breaks the host GTK stack on distros with a recent glibc/GTK (Arch, and per #29 also Debian).

What I found after --appimage-extract:

- The bundle contains only the Flutter app plus libapprun_hooks.so. GTK, gdk-pixbuf, and glibc all come from the host.
- AppRun.env sets GTK_EXE_PREFIX, GTK_DATA_PREFIX, and prepends the AppDir to XDG_DATA_DIRS, and AppRun LD_PRELOADs path-remapping hooks. On modern systems this redirects host gdk-pixbuf away from its real loaders/data, so it can't decode even GTK's built-in gresource icons, and GTK aborts:
Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495 ... Failed to load /org/gtk/libgtk/icons/32x32/status/image-missing.png: Unrecognized image file format
- Proof: launching the inner binary with a clean environment works perfectly. This also explains why overriding XDG_DATA_DIRS helped in #29.

Working workaround (the extra symlink is needed because appimage-builder patches the binary's ELF interpreter to the relative path lib64/ld-linux-x86-64.so.2):

```bash
./LinuxPowerToys-0.9-x86_64.AppImage --appimage-extract
mv squashfs-root LinuxPowerToys
mkdir LinuxPowerToys/lib64
ln -s /lib64/ld-linux-x86-64.so.2 LinuxPowerToys/lib64/
cd LinuxPowerToys && ./usr/bin/linuxpowertoys
```

(The cd matters: the relative interpreter path resolves against the working directory.)

Suggested fix: repackage withxdeploy) instead ofappimage-builder, or drop the GTK_EXE_PREFIX/GTK_DATA_PREFIX/XDG_DATA_DIRS overrides from AppRun.env. Since the bu, pointing those at the AppDironly breaks the host libraries the app depends on. appimage-builder is unmaintained and this failure mode is common flease distros.
